tests/lib/snaps/test-snapd-ssh-keys/meta/snap.yaml

```diff
@@ -0,0 +1,13 @@
+name: test-snapd-ssh-keys
+version: 1.0
+summary: Basic ssh-keys snap
+description: A basic snap which access to ssh
+
+apps:
+  ssh:
+    command: bin/ssh
+    plugs: [ssh-keys,network-control]
```

Comment:
I'm worried about network-control here again (see the review of the other ssh test)